Formula/podman.rb

```diff
@@ -56,12 +90,39 @@ def install
       libexec.install "bin/gvproxy"
     end
 
-    if build.head?
-      system "make", "podman-remote-#{os}-docs"
-    else
-      system "make", "install-podman-remote-#{os}-docs"
+    resource("podman-qemu").stage do
+      ENV["LIBTOOL"] = "glibtool"
+      args = %W[
+        --prefix=#{libexec}
+        --cc=#{ENV.cc}
+        --host-cc=#{ENV.cc}
+        --disable-bsd-user
+        --disable-guest-agent
+        --enable-curses
+        --enable-libssh
+        --enable-slirp=system
+        --enable-vde
+        --enable-virtfs
+        --extra-cflags=-DNCURSES_WIDECHAR=1
+        --disable-sdl
+        --target-list=aarch64-softmmu,x86_64-softmmu
+      ]
+      # Sharing Samba directories in QEMU requires the samba.org smbd which is
+      # incompatible with the macOS-provided version. This will lead to
+      # silent runtime failures, so we set it to a Homebrew path in order to
+      # obtain sensible runtime errors. This will also be compatible with
+      # Samba installations from external taps.
+      args << "--smbd=#{HOMEBREW_PREFIX}/sbin/samba-dot-org-smbd"
```

Comment:
Otherwise this won't work if people unlink samba.
```suggestion
      args << "--smbd=#{Formula["samba"].opt_sbin}/samba-dot-org-smbd"
```